test/services/LibraryService.test.ts

```diff
@@ -13,27 +13,56 @@ const LIBRARY_WITH_URL: fhir4.Library = {
   url: 'http://example.com'
 };
 
-const LIBRARY: fhir4.Library = {
+const LIBRARY_WITH_NO_DEPS: fhir4.Library = {
   resourceType: 'Library',
+  id: 'testLibraryWithNoDeps',
+  status: 'active',
+  url: 'http://example.com/testLibrary',
+  type: { coding: [{ code: 'logic-library' }] }
+};
+
+const LIBRARY_WITH_DEPS_2: fhir4.Library = {
+  resourceType: 'Library',
+  id: 'testLibraryWithDeps2',
+  status: 'draft',
+  url: 'http://example.com/testLibraryWithDeps2',
   type: { coding: [{ code: 'logic-library' }] },
-  id: 'test',
-  status: 'active'
+  relatedArtifact: [
+    {
+      type: 'depends-on',
+      resource: 'http://example.com/testLibrary'
+    }
+  ]
+};
+
+const LIBRARY_WITH_DEPS: fhir4.Library = {
```

Comment:
This is minor but maybe we can consider renaming `LIBRARY_WITH_DEPS` to `LIBRARY_WITH_NESTED_DEPS` and rename `LIBRARY_WITH_DEPS_2` to `LIBRARY_WITH_DEPS`. This renaming makes it easier for the developer to quickly see why we have multiple library fixtures with dependencies

Reply:
Good suggestion!